models/migrate.go

```diff
@@ -196,12 +203,11 @@ func InsertReleases(rels ...*Release) error {
 			return err
 		}
 
-		for i := 0; i < len(rel.Attachments); i++ {
+		for i, a := range rel.Attachments {
 			rel.Attachments[i].ReleaseID = rel.ID
-		}
-
-		if _, err := sess.NoAutoTime().Insert(rel.Attachments); err != nil {
-			return err
+			if _, err := sess.NoAutoTime().Insert(a); err != nil {
```

Comment:
This is not completely the same. I expect that Insert([]Attachment) will actually attempt a single insert with all the values in it whereas this code will create a separate insert statement for each attachement.

I think you need to switch this back.